userprofile/models.py

```diff
@@ -140,3 +140,13 @@ def _course_instance(self):
     @cached_property
     def _course(self):
         return self._course_instance.course
+
+
+class LTIServiceUser(GraderUser):
+    def __init__(self, submission=None, exercise=None, lti_service=None, **kwargs):
+        self._lti_service = lti_service
+        super().__init__(submission=submission, exercise=exercise, **kwargs)
+
+    @property
+    def lti_service(self):
```

Comment:
No need to have property for this. Let's change it later if needed. We any case can't protect against non accidental writes.

Reply:
Do you really want to remove the whole property? It causes no harm and it is a natural way to read the lti_service in the public API.

Reply:
I mean the property part is not needed. `_lti_service` can be renamed to `lti_service` as the property wrap doesn't bring anything extra.